infrastructure/subjects/subjects.go

```diff
@@ -0,0 +1,28 @@
+package subjects
+
+import (
+	"errors"
+
+	"github.com/exlibris-fed/exlibris/model"
+	"github.com/jinzhu/gorm"
+)
+
+var (
+	ErrNotFound = errors.New("Subject could not be found for book")
+)
+
+func New(db *gorm.DB) *Repository {
+	return &Repository{db: db}
+}
+
+type Repository struct {
+	db *gorm.DB
+}
+
+func (r *Repository) GetByID(id string) ([]*model.Subject, error) {
```

Comment:
Do we even really care about subjects?

Reply:
It's not really used at the moment, but books have a list of subjects and it's a handy bit of information to do some stuff in the future around suggestions